rules/atlassian_rules/user_logged_in_as_user.py

```diff
@@ -0,0 +1,27 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return (
+        deep_get(event, "attributes", "action", default="<unknown-action>")
+        == "user_logged_in_as_user"
+    )
+
+
+def title(event):
+    actor = deep_get(event, "attributes", "actor", "email", default="<unknown-email>")
+    impersonated_user = deep_get(
+        event, "attributes", "context", default="<unknown-user-attributes>"
+    )[0]["attributes"].get("email")
```

Comment:
```suggestion
    context = deep_get(
        event, "attributes", "context", default=[])
    context_attributes = context[0].get('attributes', {}) if context else {}
    impersonated_user = context_attributes.get('email', '<unknown-email>')
```

Reply:
something like this would handle errors if context doesn't exist or doesn't contain expected fields

Reply:
I see, this makes sense. Thank you.